aasemble/django/apps/buildsvc/urls.py

```diff
@@ -6,6 +6,7 @@
     url(r'^sources/(?P<source_id>\d+|new)/', aasemble.django.apps.buildsvc.views.package_source, name='package_source'),
     url(r'^sources/', aasemble.django.apps.buildsvc.views.sources, name='sources'),
     url(r'^repositories/', aasemble.django.apps.buildsvc.views.repositories, name='repositories'),
+    url(r'^builds/(?P<source_id>\d+)/', aasemble.django.apps.buildsvc.views.rebuild, name='rebuild'),
```

Comment:
I think the url needs to be `url(r'^builds/rebuild/(?P<source_id>\d+)/'` to conform to REST conventions. Essentially you're saying go to rebuild task and rebuild source with source_id that is given.